### ISSUE TITLE
Powerapp on mobile not working

### ISSUE THREAD

**Issue by indcoder** [amynic/ainights-sessionowners#9]
The onSelect action on the camera image does not work on the mobile app while it works on the browser. So this may not be an issue with the workshop itself but more likely, a bug in the mobile Powerapp. I have logged this issue so that other organizers may be aware of it, if they encounter this bug.

**Comment by amynic** [comment]
Good spot - did I see somewhere you mentioned your using Android?

**Comment by indcoder** [comment]
Yes ...but this could be a one-off issue plaguing a particular Android brand. More data from an heterogeneous mobile base would be needed to validate my case.
